### ISSUE TITLE
🐛 BUG: Issue with doing direct upload of sveltekit project with wrangler2

### ISSUE THREAD

**Issue by bhvngt** [cloudflare/wrangler2#1087]
### What version of `Wrangler` are you using?

2.0.6

### What operating system are you using?

Mac

### Describe the Bug

I have a monorepo project with following dir structure. Under apps/web I have a [sveltekit](https://kit.svelte.dev) project

```
\
- apps
   - web
- libs
- ....
```

When I build my project using the `pages` dashboard with following settings, everything works well.
```
Build output directory: /apps/web/.svelte-kit/cloudflare
Root directory: /
```

However if I try to publish by project using newly launched direct upload feature by executing following command
```
apps/web ➜ wrangler pages publish .svelte-kit/cloudflare/ --project-name <name>
```

While wrangler successfully uploads my project, on the published url, images are not loading - they have been kept under .svelte-kit/cloudflare/images folder. The generated `layout.css` file is getting downloaded with blank response. Though I can see valid file under `.svelte-kit/cloudflare/_app/assets/pages/__layout.svelte-94e38b0d.css`.

Am I missing something here?

**Comment by jhandsCF** [comment]
Hey there! Thanks for reporting this, we're looking into a fix. In the meantime, you can try to cd to the directory before uploading:
```
cd .svelte-kit/ && wrangler pages publish ./cloudflare --project-name <name>
```

**Comment by bhvngt** [comment]
Thanks a ton @jhandsCF. This works for me and saves a bunch of time for me.

**Comment by GregBrimble** [comment]
This should be fixed as of [2.0.15](https://github.com/cloudflare/wrangler2/pull/1284).
